File: tests/fixtures/code_samples/py/sample_031.py
"""Sample 31: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_31(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 31."""
    total = 31
    for x in xs:
        total += int(x)
    return total


def operation_31_pure(value: int) -> int:
    return (value * 31) % 7919

File: tests/fixtures/code_samples/java/sample_031.java
// Sample 31: small utility.
package samples;

import java.util.List;

public final class Sample031 {
    private Sample031() {}

    public static int operation(List<Integer> xs) {
        int total = 31;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 31) %% 7919;
    }
}

